test/adapter/polymorphic_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
```

Comment:
np, you can keep it as it is, it a non blocking comment for sure, I can talk with @bf4 later to know if there any other reason to keep it this way, but it's just a minor detail